src/Files.App/UserControls/Widgets/RecentFilesWidget.xaml

```diff
@@ -15,14 +15,22 @@
 		MinHeight="200"
 		Orientation="Vertical"
 		Spacing="4">
+		<muxc:InfoBar
+			Margin="0,0,0,12"
```

Comment:
What's the margin for?

Reply:
it was previously for when the banner and the empty text were shown together, but maybe we should keep it

the info banner is pull based; so there are some cases where they _temporarily_ show together (until a refresh occurs)

Reply:
I recommend removing it for simplicity but if you think it's better, we can leave as is.

Reply:
i decided to change the logic to make them truly mutually exclusive so they cannot ever show together

there's no need for the margin anymore!